x/oracle/keeper/keeper.go

```diff
@@ -43,3 +44,18 @@ func NewKeeper(
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
+
+func (k Keeper) AddOracleRegistrationVoteQueue(ctx sdk.Context, uniqueID string, addr sdk.AccAddress, endTime time.Time) {
+	store := ctx.KVStore(k.storeKey)
+	store.Set(types.GetOracleRegistrationVoteQueueKey(uniqueID, addr, endTime), addr)
+}
+
+func (k Keeper) GetEndOracleRegistrationVoteQueueIterator(ctx sdk.Context, endTime time.Time) sdk.Iterator {
```

Comment:
Would be better to name this as `GetClosedOracleRegistrationVoteIterator`, because the adjective `end` means `final`.